src/users/views.py

```diff
@@ -135,10 +137,48 @@ def get_object(self):
 
 
 class MentorsSearchView(generics.ListAPIView):
-    queryset = Mentor.objects.all()
+    """
+    View for finding a mentor by major
+    """
+    queryset = Mentor.objects.all().filter(active=True)
     serializer_class = MentorSerializer
 
     def filter_queryset(self, queryset):
         major = self.request.query_params['major']
         return queryset.filter(major__name=major)
 
+
+class MentorView(APIView):
+    """
+    Currently Unneeded: View Mentor By ID
```

Comment:
what does Currently Unneeded mean?  Like, if it's not specified in the API, don't put it here, and if it is, that you can remove the "Currently Unneeded".

Reply:
also, you can make this using a `generics.RetrieveAPIView`